src/js/components/tasks/table.tsx

```diff
@@ -50,13 +50,17 @@ const NameTableCell = ({
   repositorySlug,
   projectSlug,
   item,
+  className,
```

Comment:
dang it, I was so close. I had the line below (60-63) but didn't add the `className` here.